feat(tokenholder): Updates to tokenholder section and Understanding UMA (#289)
UMAprotocol/docs: docs/uma-tokenholders/uma-holders.md

---
title: Tokenholders Guide
sidebar_label: Tokenholders Guide
---

Welcome to the UMA Tokenholder guide. This page provides links for frequently used resources related to UMA tokenomics, voting, and approved price identifiers and collateral currencies.

> ### UMA Token Links

Reviewer: ```suggestion
### Links
```

The links below walk through the tokenomics of the UMA token and voting process:

- [UMA Tokenomics](/uma-tokenholders/uma-tokenomics)
- [Voter dApp](https://vote.umaproject.org/)

> ### Price Identifier and Collateral Links

Reviewer: ```suggestion
### Governance Links
```

> The links below are resources realted to price identifiers and collateral currencies approved by UMA tokenholders:

Reviewer: ```suggestion
The links below are resources related to participating in UMA governance, primarily detailing price identifiers and collateral currencies already approved by UMA tokenholders, as well as how one could participate in the UMA Improvement Proposal (UMIP) process:
```
